packages/remote-config/android/src/reactnative/java/io/invertase/firebase/config/ReactNativeFirebaseConfigModule.java

```diff
@@ -255,6 +255,20 @@ public void removeConfigUpdateRegistration(String appName) {
     }
   }
 
+  @ReactMethod
+  public void setCustomSignals(String appName, ReadableMap setCustomSignals, Promise promise) {
+    module
+      .setCustomSignals(appName, setCustomSignals.toHashMap())
```

Comment:
tiniest quibble, but the param name seems odd, suggest calling it customSignals (or customSignalsMap but that seems redundant, so customSignals)

```suggestion
  public void setCustomSignals(String appName, ReadableMap customSignals, Promise promise) {
    module
      .setCustomSignals(appName, customSignals.toHashMap())
```

Reply:
Yeah, that was a mistake, good catch 👍